README.md

```diff
@@ -1,49 +1,192 @@
+To run `staramrnf`, you will need to include both mandatory parameters:
+
+#### Mandatory Parameters
 
-The input to the pipeline is a standard sample sheet (passed as `--input samplesheet.csv`) that looks like:
+- `--input`: a URI to the samplesheet
+- `--output`: the directory for pipeline output
 
-| sample  | contigs          | species             |
-| ------- | ---------------- | ------------------- |
-| SampleA | genomeA.fastq.gz | Salmonella enterica |
+```bash
+nextflow run phac-nml/staramrnf -r main -latest -profile docker --outdir path/output_folder --input path/samplesheet.csv
+```
 
-The structure of this file is defined in [assets/schema_input.json](assets/schema_input.json). Validation of the sample sheet is performed by [nf-validation](https://nextflow-io.github.io/nf-validation/).
+## Input
 
-### Output
+### Samplesheet Input
+
+You will need to create a samplesheet with information about the samples you would like to analyze before running the pipeline. Use this parameter to specify its location.
 
 ```bash
-results/staramr/
-└── SampleA_results
-    ├── detailed_summary.tsv
-    ├── mlst.tsv
-    ├── plasmidfinder.tsv
-    ├── resfinder.tsv
-    ├── results.xlsx
-    ├── settings.txt
-    └── summary.tsv
+--input '[path to samplesheet file]'
+```
+
+### Samplesheet Description
+
+The input samplesheet requires two columns: `sample`, `contigs` with an optional third column `species`. The `species` column is used in the selecting of the Pointfinder organism database (empty if "None"). Rows of the `sample` column within a samplesheet must be unqiue. All other columns will be ignored.
+
+Note: The [parameter](#-Parameters) `--pointfinder_database` overrides the `species` column for all samples.
+
+A final samplesheet file consisting of `sample`, `contigs` and `species`.
+
+```csv title="samplesheet.csv"
+sample,contigs,species
+SAMPLE1,sample1.fastq.gz,Salmonella
+SAMPLE2,sample2fastq.gz,Escherichia coli
+SAMPLE3,sample3.fastq.gz,
+```
+
+| Column    | Description                                                                                          |
+| --------- | ---------------------------------------------------------------------------------------------------- |
+| `sample`  | Custom sample name. Samples should be unique within a samplesheet. **Required**                      |
+| `contigs` | Full path to genome contig(s). Uncompressed or gzipped (.gz) fasta file (fna,fa,fasta). **Required** |
+| `species` | Species of genome (see accepted Pointfinder organisms below). **Optional**                           |
+
+An [example samplesheet](../assets/samplesheet.csv) has been provided with the pipeline.
+
+Note: Validated Pointfinder organisms for `species` include: Enterococcus faecalis, Helicobacter pylori, Salmonella, Enterococcus faecium, Escherichia coli, Campylobacter. Accepted but unvalidated species: Klebsiella, Staphylococcus aureus, Mycobacterium tuberculosis, Neisseria gonorrhoeae, Plasmodium falciparum.
+
+## Output
```

Comment:
Can you provide a link to `docs/output.md` in this section.

Reply:
d5ae28d